jobs/periodic/redact-inactive-accounts/send-inactivity-notification.ts

```diff
@@ -0,0 +1,27 @@
+import * as Notification from '../../../common/notification';
+import { Person } from '../../../common/notification/types';
+import moment from 'moment';
+import { findAllPersons } from './query';
+import { getLogger } from '../../../common/logger/logger';
+
+export const NOTIFY_AFTER_DAYS = 3 * 365 + 9 * 30; // 3 years and 9 months
+
+const logger = getLogger('SendInactivityNotification');
+
+export async function sendInactivityNotification() {
+    logger.info('Start sending inactivity notification');
+
+    const p = await findAllPersons(true, moment().startOf('day').subtract(NOTIFY_AFTER_DAYS, 'days').toDate());
```

Comment:
There is still a bug, which would create one notification every day. We need to make sure that we only send it once.

Reply:
I've added an integration test for this case, which fails right now.

Reply:
Solved by adding a uniqueID for each notification